gnovm/cmd/gno/bug.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"net/url"
+	"os/exec"
+	"runtime"
+	"strings"
+	"text/template"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+const bugTmpl = `## [Subject of the issue]
+
+### Description
+
+Describe your issue in as much detail as possible here
+
+### Your environment
+
+* go version {{.GoVersion}} {{.Os}}/{{.Arch}}
+* version of gno
+* branch that causes this issue (with the commit hash)
+
+### Steps to reproduce
+
+* Tell us how to reproduce this issue
+* Where the issue is, if you know
+* Which commands triggered the issue, if any
+
+### Expected behaviour
+
+Tell us what should happen
+
+### Actual behaviour
+
+Tell us what happens instead
+
+### Logs
+
+Please paste any logs here that demonstrate the issue, if they exist
+
+### Proposed solution
+
+If you have an idea of how to fix this issue, please write it down here, so we can begin discussing it
+
+`
+
+type bugCfg struct {
+	skipBrowser bool
+}
+
+func newBugCmd(io commands.IO) *commands.Command {
+	cfg := &bugCfg{}
+	return commands.NewCommand(
+		commands.Metadata{
+			Name:       "bug",
+			ShortUsage: "bug",
+			ShortHelp:  "Start a bug report",
+		},
+		cfg,
+		func(_ context.Context, args []string) error {
+			return execBug(cfg, args, io)
+		},
+	)
+}
+
+func (c *bugCfg) RegisterFlags(fs *flag.FlagSet) {
+	fs.BoolVar(
+		&c.skipBrowser,
+		"skip-browser",
+		false,
+		"do not open the browser",
+	)
+}
+
+func execBug(cfg *bugCfg, args []string, io commands.IO) error {
+	if len(args) != 0 {
+		return flag.ErrHelp
+	}
+
+	bugReportEnv := struct { // TODO: include gno version or commit?
+		Os, Arch, GoVersion string
+	}{
+		runtime.GOOS,
+		runtime.GOARCH,
+		runtime.Version(),
+	}
+
+	var buf strings.Builder
+	tmpl, err := template.New("bug.tmpl").Parse(bugTmpl)
+	if err != nil {
+		return err
+	}
+	tmpl.Execute(&buf, bugReportEnv)
+
+	body := buf.String()
+	url := "https://github.com/gnolang/gno/issues/new?body=" + url.QueryEscape(body)
+
+	if !cfg.skipBrowser {
+		// Try opening browser (ignore error)
+		_ = openBrowser(url)
```

Comment:
mhhh. don't print the template if opening browser succeeds?

Reply:
i just did the opposite 🤷‍♂️

https://github.com/gnolang/gno/pull/1325#discussion_r1388140728
> ...
> You can always print the issue body in the console, regardless if the browser opened or not

Reply:
Done: eb277078e1cf1f040e869b36c11fd45ae4c9bbe3